src/style.css

```diff
@@ -1 +1,63 @@
 /* add styles here */
+
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: 300;
+}
+
+.search-bars {
+  box-sizing: border-box;
+  display: flex;
+  width: 100%;
+  margin-top: 20px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+}
+
+.search-bars::placeholder {
+  font-family: Avenir, sans-serif;
+  color: #3d4e61;
```

Comment:
The `font-family` property should have a fallback font. If the `Avenir` font fails to load for some reason, the browser will use the fallback font. You can add a generic family name like `serif` or `sans-serif` as a fallback. For example: `font-family: Avenir, sans-serif;`

Reply:
IT'S ALREADY DONE